lemur/plugins/lemur_aws/plugin.py

```diff
@@ -750,3 +750,85 @@ def send(self, notification_type, message, excluded_targets, options, **kwargs):
 
         current_app.logger.info(f"Publishing {notification_type} notification to topic {topic_arn}")
         sns.publish(topic_arn, message, notification_type, options, region_name=self.get_option("region", options))
+
+
+class AWSACMSourcePlugin(SourcePlugin):
+    title = "AWS-ACM"
+    slug = "aws-acm-source"
+    description = "Discovers all ACM SSL certificates in an AWS account"
```

Comment:
```suggestion
    description = "Discovers all ACM TLS certificates in an AWS account"
```
nit: since it's been deprecated since 1999

Reply:
Done!